packages/aws-cdk/lib/api/deploy-stack.ts

```diff
@@ -239,6 +243,17 @@ export async function deployStack(options: DeployStackOptions): Promise<DeploySt
   } else {
     print('Changeset %s created and waiting in review for manual execution (--no-execute)', changeSetName);
   }
+
+  // Update termination protection only if it has changed.
+  if (cloudFormationStack.terminationProtection !== terminationProtection) {
+    debug('Updating termination protection for stack %s', deployName);
```

Comment:
add the previous and new values of `terminationProtection` in this log

Reply:
done